src/fu-history.c

```diff
@@ -307,6 +311,28 @@ fu_history_migrate_database_v5(FuHistory *self, GError **error)
 	return TRUE;
 }
 
+static gboolean
+fu_history_migrate_database_v6(FuHistory *self, GError **error)
+{
+	gint rc;
+	rc = sqlite3_exec(self->db,
+			  "CREATE TABLE IF NOT EXISTS hsi_history ("
+			  "last timestamp TIMESTAMP DEFAULT CURRENT_TIMESTAMP,"
```

Comment:
out of interest, what's the `last` keyword here?

Reply:
It's a column name.

Reply:
Ohh, I think you might be copying my typo of `created timestamp TIMESTAMP DEFAULT CURRENT_TIMESTAMP` -- I think the `timestamp` is probably duplicated for no reason. I think you probably want `timestamp TIMESTAMP...` for the column to be called `timestamp`.

Reply:
You are right!  Let me correct it.